src/apps/backend/modules/otp/internal/otp-util.ts

```diff
@@ -21,4 +24,18 @@ export default class OtpUtil {
     }
     return otp;
   }
+
+  public static getOtp(phoneNumber: string): string {
+    const isProdEnv = process.env.NODE_CONFIG_ENV === 'production';
+
+    // If the phone number is the default phone number in production or if we are in a
+    // non-production environment and the default OTP is set in the config, return the default OTP
+    if (
+      (isProdEnv && isDefaultPhoneNumber(phoneNumber))
+      || (!isProdEnv && ConfigService.hasValue('defaultOTP'))
+    ) {
+      return ConfigService.getValue<string>('defaultOTP');
+    }
+    return OtpUtil.generateOtp(OTP_LENGTH);
```

Comment:
For now let's just simplify this to:
- If default OTP and phone number is configured and phone number matches with the config, return default OTP.
- Otherwise generate OTP
- Remove any env based checks

Reply:
Simplified it.

Reply:
There are few things I'm concerned about using this approach:
- In development env either we can only login using `otp.defaultPhoneNumber` and `otp.defaultOTP` set in development config or we need to view OTP from local DB and use it to login.
- The above goes same for the preview env since `sms.enabled` will be set to `false` in both env.

As of Now we can Login/Create account using any Number by entering `otp.defaultOTP` set in config for development env and preview env as well.
Let me know if this works.

Reply:
Ideally:
- In development - As the sms is disabled, you should get the OTP from console itself.
- In preview - I have request let's enable SMS there

Will this works?